cost-analyzer/charts/prometheus/templates/server-deployment.yaml

```diff
@@ -59,10 +59,15 @@ spec:
             - --volume-dir={{ . }}
           {{- end }}
           resources:
-{{ toYaml .Values.configmapReload.prometheus.resources | indent 12 }}
-          {{- with .Values.configmapReload.prometheus.containerSecurityContext }}
+          {{- toYaml .Values.configmapReload.prometheus.resources | nindent 12 }}
           securityContext:
-            {{- toYaml . | nindent 12 }}
+          {{- if .Values.global.containerSecurityContext }}
+            {{- toYaml .Values.global.containerSecurityContext | nindent 12 }}
+          {{- else if .Values.global.containerSecurityContext }}
```

Comment:
Note, the `if` and `else if ` statements here look to be the same

Reply:
thanks for finding this!